src/uopool/mod.rs

```diff
@@ -18,18 +25,20 @@ use jsonrpsee::tracing::info;
 use parking_lot::RwLock;
 use std::{collections::HashMap, fmt::Debug, net::SocketAddr, sync::Arc, time::Duration};
 
-pub mod memory;
+pub mod memory_mempool;
+pub mod memory_reputation;
 pub mod server;
 pub mod services;
 
 pub type MempoolId = H256;
 
+pub type MempoolBox<T> = Box<dyn Mempool<UserOperations = T>>;
+pub type ReputationBox<T> = Box<dyn Reputation<ReputationEntries = T>>;
```

Comment:
I am curious why this needs to be `dyn` now. Why not just use `MemoryMempool` and `MemoryReputation`.

Reply:
Well, my idea was that sometimes, someone would want to use `MemoryMempool` for one mempool and `DatabaseMempool` for another. But at the moment, this is somewhat complex, yeah.

Reply:
Hm. Ok. Let's leave it as it is now.